src/zoom_type.h

```diff
@@ -35,7 +35,7 @@ enum ZoomLevel : byte {
 	ZOOM_LVL_VIEWPORT = ZOOM_LVL_OUT_4X, ///< Default zoom level for viewports.
 	ZOOM_LVL_NEWS     = ZOOM_LVL_OUT_4X, ///< Default zoom level for the news messages.
 	ZOOM_LVL_INDUSTRY = ZOOM_LVL_OUT_8X, ///< Default zoom level for the industry view.
-	ZOOM_LVL_TOWN     = ZOOM_LVL_OUT_8X, ///< Default zoom level for the town view.
+	ZOOM_LVL_TOWN     = ZOOM_LVL_OUT_4X, ///< Default zoom level for the town view.
```

Comment:
In `town_gui.cpp:334` the wrong constant was used (`ZOOM_LVL_NEWS`). This has been corrected and the constant has been updated to reflect the current zoom level for towns.